src/web/modules/agreements/status.macro.njk

```diff
@@ -0,0 +1,15 @@
+{% macro status(state) %}
```

Comment:
This looks like a good start, I'm guessing we might want to show the "created" state as grey to differentiate it from "active" in the future. I think that's a small thing that can be done in a follow up, example here https://github.com/alphagov/pay-selfservice/blob/master/app/views/agreements/macro/status.njk

Reply:
Thanks - I have copied that self service macro.